openedx_external_enrollments/tests/external_enrollments/test_greenfig_external_enrollment.py

```diff
@@ -0,0 +1,77 @@
+"""Tests EdxInstanceExternalEnrollment class file."""
+from django.test import TestCase
+from mock import Mock, patch
+
+from openedx_external_enrollments.external_enrollments.greenfig_external_enrollment import (
+    GreenfigInstanceExternalEnrollment,
+)
+
+
+class GreenfigInstanceExternalEnrollmentTest(TestCase):
```

Comment:
Any chance to test these methods: 
_execute_post
_get_download_headers
_get_course_list? @luismorenolopera

Reply:
Any reason to avoid testing test _get_course_list? @luismorenolopera